src/components/ImageUploader.js

```diff
@@ -127,13 +158,13 @@ class ImageUploader extends React.Component {
         this.notify(constants.NOT_CERTIFIED_WARNING);
         return;
       } else {
-        if (!this.state.file) {
+        if (!this.state.files) {
           this.notify(constants.NO_FILE_SELECTED_WARNING);
         }
 
-        this.checkImageDimensions(this.state.file).then(
-          ({ width, height }) => {
-            this.handleUpload(stepIndex, width, height);
+        Promise.all(this.checkImageDimensions(this.state.files)).then(
+          () => {
+            this.handleUpload(stepIndex, this.vars.widths, this.vars.heights);
           },
           () => {
             this.notify(constants.MAX_WIDTH_HEIGHT_WARNING);
```

Comment:
Now you need to specify here, which particular image failed.

Reply:
Can I do it in this way:

If all of the images are valid then passed, else reject and prompts error message "image uploaded cannot be greater than 1024" instead of prompts error message for each image.

Reply:
Actually, best way to show will be to instead in last step show which of the images passed and which one failed